### DISCUSSION TITLE
Auto-Approval for Azure DevOps integration

### DISCUSSION THREAD

**Discussion by dannyyy** [renovatebot/renovate#8303] [Request Help]
In our Azure DevOps project we have on all repositories a branch policy to protect the master branch. The policy requires at least one approval from someone who is part of a certain list. For the repository who needs auto-merge, we are able to weaken the policy to allow self-approval.

Is there a configuration setting in renovate to apply a self-approval? Actually the same as the azureAutoComplete attribute, just for the approval. Then we could implement something like:
- 1 approval required
- The bot's service-account is part of the required approval-group
- And the policy allowes self-approvals

For GitHub users I've seen that there is another integration ( github.com/apps/renovate-approve) just for that purpose.

**Comment by viceice** [comment]
No, renovate will never approve his own pr. You need something else or disable approval for renovate user account.
